site/tutorials/gettings_started.md

```diff
@@ -315,3 +321,5 @@ Now you can create a website that embeds a Vega-Lite specification. If you want
 - See the [examples gallery]({{site.baseurl}}/examples/gallery.html).
 - Build your own visualizations in the [online editor](https://vega.github.io/vega-editor/?mode=vega-lite).
 - Browse through the [documentation]({{site.baseurl}}/docs/).
+- Learn [how to manually compile Vega-Lite specifications into Vega specifications](https://vega.github.io/vega-lite/usage/compile.html).
```

Comment:
But this is NOT the main part of the tutorial, this is saying what we didn't say up there.  
A lot of people want to minimize # of dependency in production, better let them aware of an option.